src/update_sdk.cmd

```diff
@@ -0,0 +1,43 @@
+REM This script replaces the SDK folder with the latest bits from the Azure SDK repository.
+REM It removes some files we do not need.  It currently targets the develop branch until fixes are
+REM merged into master.
+
```

Comment:
Yeah I'm meeting with the core sdk engineers today to discuss how they'd like to handle updates.  It is proposed that they actually publish an arduino specific repo from their internal build system along with the core SDK, and then we replace all of this with a sub module.  If we keep the script I'll get a cross-platform implementation.